evergreen.yml

```
@@ -1362,143 +1362,129 @@ tasks:
   - name: compile
     depends_on:
     commands:
+      - func: "install iODBC"
+        variants: [macos, macos-arm]
       - func: "install rust toolchain"
+        variants: [ubuntu2204, windows-64]
       - func: "install unix odbc"
         variants: [ubuntu2204]
       - func: "set packages version"
       - func: "check packages version"
       - func: "compile release"
+        variants: [ubuntu2204, windows-64]
+      - func: "compile macos release"
+        variants: [macos, macos-arm]
       - func: "build msi"
         variants: [windows-64]
       - func: "tar linux artifacts"
         variants: [ubuntu2204]
-      - func: "mciuploads release artifacts"
-
-  - name: compile-macos
-    depends_on:
-    commands:
-      - func: "install iODBC"
-      - func: "set packages version"
-      - func: "check packages version"
-      - func: "compile macos release"
       - func: "build dmg"
+        variants: [macos, macos-arm]
       - func: "mciuploads release artifacts"
 
   - name: compile-debug
     commands:
+      - func: "install iODBC"
+        variants: [macos, macos-arm]
       - func: "install rust toolchain"
+        variants: [ubuntu2204, windows-64]
       - func: "install unix odbc"
         variants: [ubuntu2204]
       - func: "set packages version"
       - func: "check packages version"
       - func: "compile debug"
```

```suggestion
      - func: "compile ubuntu and win debug"
```
Same as above, if you think it would help with clarifying the task